semgrep/semgrep/app/session.py

```diff
@@ -43,7 +46,10 @@ def __str__(self) -> str:
         return result
 
 
-class AppSession(requests.Session):
+REQUESTS_CACHE_PATH = USER_DATA_FOLDER / "requests-cache.sqlite"
```

Comment:
TODOs with the whole caching concept:

- [ ] Make it possible to disable cache
- [ ] Make it possible to change cache path (maybe USER_DATA_FOLDER is already configurable enough?)
- [ ] Clean up expired entries when running